docs/NewFeatures/CRR_statistics.rst

```diff
@@ -0,0 +1,10 @@
+S3 Cross-Region Replication (CRR) Statistics
+--------------------------------------------
+
+This feature enables replication status tracking. The replication system
+exposes metrics through a REST API to monitor the replication of
+pending, processing, and completed objects. It returns the number of
+errors that occurred during replication, the current throughput—in
+operations per second or replicating objects per second—and the number

```

Comment:
I think "in replication operations per second" works well enough to not need the "or" here.

Reply:
Does this work?

"It returns the number of failures that occurred during replication, the current throughput—in replication operations or objects per second—and the total of bytes completing per second."

Reply:
Sounds perfect to me! @philipyoo What do you think?

Reply:
sgtm 👍